.gitlab-ci.yml

```diff
@@ -775,12 +775,14 @@ test_accep_raspberrypi4:
     reports:
       junit: results_accep_raspberrypi4.xml
 
-test_backend_integration:
+test_backend_integration_enterprise:
```

Comment:
Idea:
Is it nicer to have:
`test_backend_integration:enterprise` set?
This can then subtype the same job, based on the variable set, and all the rest of the code can be shared?

Thus you have `test_backend_integration` as a shared stage, and then
`test_backend_integration:enterprise`
and
`test_backend_integration:open` 
inherits from this ?